### PR TITLE
Fixes #33883 - Handle permissions for Ansible config jobs

### PR DESCRIPTION
**Pull request by xprazak2**
needs https://github.com/theforeman/foreman-tasks/pull/658

### CODE REVIEW

**Comment by adamruzicka** [comment]
foreman-tasks-5.2.0 containing the necessary changes was released. Could you bump the dependency here?

**Comment by xprazak2** [comment]
It seems like we do not depend on tasks directly here, but rather on rex. What rex version should I bump to?

**Comment by adamruzicka** [comment]
As far as I know rex doesn't depend on anything that was added in tasks 5.2.0, so just bumping rex dependency won't help you get the bits you need. I'd say relying on transitive dependencies is incorrect in case you explicitly need something

**Comment by xprazak2** [comment]
Good point, I added tasks.

File: webpack/components/AnsibleHostDetail/components/JobsTab/RecurringJobsTable.js [hunk 1]
```diff
@@ -35,6 +35,19 @@ const RecurringJobsTable = ({ jobs, resourceName, resourceId }) => {
     );
   };
 
+  const actionItems = job => {
+    const items = [];
+    if (job.recurringLogic.meta.canEdit) {
+      items.push({
+        title: __('Cancel'),
+        onClick: onJobCancel(job.recurringLogic.id),
```
Comment (amirfefer) [inline webpack/components/AnsibleHostDetail/components/JobsTab/RecurringJobsTable.js:43]: doesn't it suppose to run on click? 
```suggestion
        onClick: () => onJobCancel(job.recurringLogic.id),
```

**Review by amirfefer — COMMENTED** [review]
Thanks @xprazak2! overall looks good 👍

File: webpack/components/AnsibleHostDetail/components/JobsTab/RecurringJobsTable.js (hunk 1 above)
Comment (xprazak2) [reply webpack/components/AnsibleHostDetail/components/JobsTab/RecurringJobsTable.js:43]: It runs on click, see l.27.

**Comment by ezr-ondrej** [comment]
The JS tests are failing

**Comment by ares** [comment]
Who has the ball now? :-)

**Review by ezr-ondrej — APPROVED** [review]
I tried to fix the tests over the weekend, but I failed.
The failures are not related, but the failing tests would test the changed components, so we don't know if we didn't break something.
Tho I'll solve that once I manage to fix the tests :+1: 
Thanks @xprazak2 !

**Pull request merged**